src/components/shared/accountsTable/index.js

```diff
@@ -0,0 +1,135 @@
+import { DEFAULT_LIMIT } from '../../../constants/monitor';
+import styles from './accountsTable.css';
+
+class AccountsTable extends React.Component {
+  handleLoadMore = () => {
+    const { accounts } = this.props;
+    accounts.loadData({ offset: accounts.data.length });
+  }
+
+  render() {
+    const {
+      accounts,
+      emptyStateMessage,
+      isLoadMoreEnabled,
+      isMediumViewPort,
+      networkStatus,
+      t,
+      title,
+    } = this.props;
+    const supply = networkStatus.data.supply;
+
+    return (
+      <Box main isLoading={accounts.isLoading} className="accounts-box">
+        <BoxHeader>
+          <h1>{title}</h1>
+        </BoxHeader>
+        {
+          accounts.error
+            ? (
+              <BoxContent>
+                <BoxEmptyState>
+                  <Illustration name="emptyWallet" />
+                  <h3>{emptyStateMessage || `${accounts.error}`}</h3>
+                </BoxEmptyState>
+              </BoxContent>
+            )
+            : (
+              <React.Fragment>
+                <BoxContent className={styles.content}>
+                  <Table
+                    getRowLink={account => `${routes.accounts.path}/${account.address}`}
+                    data={accounts.data}
+                    columns={[
+                      {
+                        header: t('Rank'),
+                        className: `${grid['col-xs-1']} ${grid['col-md-1']}`,
+                        id: 'rank',
+                        getValue: () => (<span className={styles.counter} />),
+                      },
+                      {
+                        header: t('Address'),
+                        className: `${grid['col-xs-3']} ${grid['col-md-5']}`,
+                        id: 'address',
+                        getValue: account => (
+                          <AccountVisualWithAddress
+                            address={account.address}
+                            isMediumViewPort={isMediumViewPort}
+                            transactionSubject="address"
+                            showBookmarkedAddress
+                          />
+                        ),
+                      },
+                      {
+                        header: t('Balance'),
+                        className: `${grid['col-xs-3']} ${grid['col-md-3']}`,
+                        id: 'balance',
+                        getValue: account => (
+                          <LiskAmount val={account.balance} roundTo={0} token={tokenMap.LSK.key} />
+                        ),
+                      },
+                      {
+                        header: t('Supply'),
+                        className: `${grid['col-xs-2']} ${grid['col-md-1']}`,
+                        id: 'supply',
+                        getValue: (account) => {
+                          const amount = new BigNumber(account.balance / supply * 100);
+                          return <span>{`${amount.toFormat(2)} %`}</span>;
```

Comment:
🐛 The supply doesn't use the correct decimal separator in German locale

For an example of implementation, refer to https://github.com/LiskHQ/lisk-desktop/blob/c156288dbf5502e56bfe5153849ed1463147275d/src/components/shared/delegatesTable/index.js#L40